src/guide/misc-using-binaries.md

```diff
@@ -0,0 +1,173 @@
+
+::: info
+
+Listed examples only reference the `dev` versions of the Docker images.
+
+For a list of all available Docker images for Iroha 2, see the [Iroha 2 Docker Hub](https://hub.docker.com/r/hyperledger/iroha2) page.
+
+:::
+
+## Using the Source GitHub Repository {#source}
+
+To perform any of the actions with the GitHub repository (i.e., [building & installing](#source-install), or [running](#source-run) binaries), the [Rust toolchain](https://www.rust-lang.org/) must first be installed. The toolchain is delivered with the package manager—[Cargo](https://doc.rust-lang.org/cargo/index.html)—that allows for installation of Rust crates from different sources, including GitHub.
+
+The recommended way to install the Rust toolchain is through a Rust installer and version management tool—[Rustup](https://rust-lang.github.io/rustup/) (See also: [Install Rust](https://www.rust-lang.org/tools/install))—by executing the following command in your terminal:
+
+  ```bash
+  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+  ```
+
+::: info
+
+Depending on the operating system that you are using or other limitations, you may want to opt for a different installation method.
+
+For all available installation methods, see the [Rust Forge: Other Rust Installation Methods](https://forge.rust-lang.org/infra/other-installation-methods.html) article.
+
+In case you experience any issues with Rust installation, see our [Troubleshooting Rust Toolchain](/guide/troubleshooting/installation-issues#troubleshooting-rust-toolchain) topic.
+
+:::
+
+### Installing from Source GitHub Repository {#source-install}
+
+To perform a system-wide installation of an Iroha 2 binary with `cargo install`, execute one of the following commands in your terminal:
+
+::: code-group
+
+```shell [Command template]
+cargo install [options] --git [repo url] --branch [branch name] [binary name]
+```
+
+```shell [iroha]
+cargo install --git https://github.com/hyperledger/iroha.git --branch iroha2-dev iroha
+
+iroha --help
+```
+
+```shell [iroha_client_cli]
+cargo install --git https://github.com/hyperledger/iroha.git --branch iroha2-dev iroha_client_cli
+
+iroha_client_cli --help
+```
+
+```shell [kagami]
+cargo install --git https://github.com/hyperledger/iroha.git --branch iroha2-dev kagami
+
+kagami --help
+```
+
+:::
+
+::: info
+
+Listed examples only reference the `dev` versions of the Iroha 2 binaries.
+
+To install a different version of a binary, specify the corresponding branch (i.e., `iroha2-lts` or `iroha2-stable`) when executing the `cargo install` command.
+
+:::
+
+::: tip
+
+For more details on `cargo install` and its `[options]`, see [The Cargo Book > cargo install](https://doc.rust-lang.org/cargo/commands/cargo-install.html).
+
+:::
+
+### Running from Cloned GitHub Repository {#source-run}
+
+First, clone the [hyperledger / iroha](https://github.com/hyperledger/iroha.git) GitHub repository, then checkout the required branch (`dev`, `lts`, or `stable`):
```

Comment:
It is usually written without spaces. (I mean, any repo on GitHub)

```suggestion
First, clone the [hyperledger/iroha](https://github.com/hyperledger/iroha.git) GitHub repository:
```

Reply:
I think for now we need to mention the branch checkout (it's part of the code below anyway)